.travis.yml

```diff
@@ -1,12 +1,5 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
```

Comment:
@hhamon the Symfony used an explicit matrix entirely just because of some experiments with a different order of jobs (running component jobs first), which was then reordered without going back to the simpler way to specify versions